tests/models/test_models.py

```diff
@@ -42,6 +42,7 @@ def test_model_original_and_dirty_attributes(self):
         self.assertEqual(model.get_original("username"), "joe")
         self.assertEqual(model.get_dirty("username"), "bob")
         self.assertEqual(model.__dirty_attributes__["username"], "bob")
+        self.assertEqual(list(model.get_dirty_keys()), ["username"])
```

Comment:
Should we make the keys return a list by default. I believe the way you have it right now it will return a dict_keys() structure. Which is why you're doing this weird thing in the test